controllers/disruption_controller.go

```diff
@@ -555,10 +538,9 @@ func (r *DisruptionReconciler) createChaosPods(instance *chaosv1beta1.Disruption
 		// get IDs of targeted containers or all containers
 		targetContainers, err = chaosv1beta1.TargetedContainers(pod, instance.Spec.Containers)
 		if err != nil {
-			dErr := chaostypes.DisruptionError{Err: fmt.Errorf("error getting target pod container ID: %w", err)}
-			dErr.AddContext("targetPodStatus", pod.Status.String())
-			dErr.AddContext("targetPodName", target)
-			dErr.AddContext("targetPodNamespace", instance.Namespace)
+			dErr := fmt.Errorf("error getting target pod's container ID: %w", err)
```

Comment:
This `err` already contains the target pod's name and container name. The namespace is already contained in the disruption logs. Additionally, the `Status.String()` was not printing something as simple as "Running" or "Completed", but an enormous struct, and thus not useful